src/constants/memory/wram.asm

```diff
@@ -1314,8 +1314,8 @@ wC50D::
 wC50E::
   ds 1 ; C50E
 
-; Unlabeled
-wC50F::
+; Entity index of Marin (as follower on the bridge on Tal Tal Heights)
```

Comment:
As follower, OR on the birdge on taltal heights. And maybe in more situations, not sure.

Reply:
Yes, I think it is also used during the short cutscene when Marin leaves Link to go to the Animal's Village.

I updated the comment.